src/mvAppItem.h

```diff
@@ -314,26 +314,28 @@ GetEntityCommand(mvAppItemType type)
     case mvAppItemType::mvInputFloatMulti:             return "add_input_floatx";
     case mvAppItemType::mvInputDoubleMulti:            return "add_input_doublex";
     case mvAppItemType::mvDragPoint:                   return "add_drag_point";
+    case mvAppItemType::mvDragRect:                    return "add_drag_rect";
     case mvAppItemType::mvDragLine:                    return "add_drag_line";
     case mvAppItemType::mvAnnotation:                  return "add_plot_annotation";
+    case mvAppItemType::mvTag:                         return "add_plot_tag";
     case mvAppItemType::mvLineSeries:                  return "add_line_series";
     case mvAppItemType::mvScatterSeries:               return "add_scatter_series";
     case mvAppItemType::mvStemSeries:                  return "add_stem_series";
     case mvAppItemType::mvStairSeries:                 return "add_stair_series";
     case mvAppItemType::mvBarSeries:                   return "add_bar_series";
+    case mvAppItemType::mvGroupBarSeries:              return "add_group_bar_series";
```

Comment:
`add_group_bar_series`: To me, **_add_bar_group_series_** sounds better / more correct (since it's a graph consisting of bar groups). It is also closer to the name of ImPlot function, `PlotBarGroups`.